README.md

```diff
@@ -6,9 +6,11 @@ Add ability to run RSpec and see the output without leaving Atom.
 
 HotKeys:
 
-- __Ctrl+Alt+T__ - executes all specs the current file
-- __Ctrl+Alt+X__ - executes only the spec on the line the cursor's at
-- __Ctrl+Alt+E__ - re-executes the last executed spec
+- <kbd>ctrl</kbd> + <kbd>alt</kbd> + <kbd>T</kbd> - executes all specs the current file
+- <kbd>ctrl</kbd> + <kbd>alt</kbd> + <kbd>X</kbd> - executes only the spec on the line the cursor's at
+- <kbd>ctrl</kbd> + <kbd>alt</kbd> + <kbd>E</kbd> - re-executes the last executed spec
```

Comment:
The `<kbd>` tags are more visually appealing, and also improve reading accessibility for users with visual disabilities